packages/plugins/plugin-mixpanel/src/methods/__tests__/identify.test.ts

```diff
@@ -0,0 +1,186 @@
+import identify from '../identify';
+import type {
+  SegmentMixpanelSettings,
+  IdentifyEventType,
+} from '@segment/analytics-react-native';
+import { mockIntegrationSettings } from '../__mocks__/mockIntegrationSettings';
+import { Mixpanel } from '../__mocks__/mixpanel-react-native';
+
+describe('#identify', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('calls identify with userId', () => {
+    const payload = {
+      type: 'identify',
+      traits: {
+        firstName: 'John',
+        phone: '(555) 555-5555',
+        foo: 'bar',
+      },
+      userId: 'user',
+    } as IdentifyEventType;
```

Comment:
nit: DRY by putting the default stuff in a const and using destructuring to replace the values you need for each test. Like:
```
const defaultPayload = {
        type: 'identify',
        traits: {
          firstName: 'John',
          phone: '(555) 555-5555',
          foo: 'bar',
        },
        userId: 'user',
      } as IdentifyEventType;
// ... then inside the test
const payload = { ...defaultPayload, userId: undefined }
```

Reply:
I can do this once I figure out how to clean up the duplicated code with `beforeEach()`